src/release/triage-procedure.md

```diff
@@ -247,9 +247,44 @@ future work.
 
 ## Issue triage
 
-Issue triage is mostly much simpler. After finishing PR triage, go to the [list
-of untagged issues] and add tags as you see fit. The following categories
-should, ideally, be assigned to each issue:
+### Incorrectly Filed Issues
+
+Frequently, issues are filed against the wrong repository. In that case, you can
+use GitHub's "Transfer issue" function to move it to the right place. When you
+do that, please also leave a comment explaining why you moved it.
```

Comment:
We should direct people to the triagebot issue transfer feature, which requires a bit less permissions, although it does require the source repository to have it set up.